docs/proposals/scheduler/README.md

```diff
@@ -77,7 +77,9 @@ Every events handled by the watcher, they are all either coming from node or `sc
 
 ### Recovery in case of crash
 
-As written above, `scheduler` is a SPOF, this part is about how we can manage a recovery. 
+Watcher self re-instanciate himself and get last data/metrics saved from etcd.
```

Comment:
some english fixes:

Watcher reinstanciates itself and gets last metrics and data from `etcd`.
etcd makes sense because it provides key/value storage, potentially with role-based access ( `leaders` can read, write and update, while others can only read). It provides safe cold storage on disk and great performance, handles on-change events (useful for our watcher or controller ?). `etcd` can be read by all server components, so we can get state metrics directly from the controller or scheduler, if that makes sense.